### PR TITLE
Currency: Fix overtriggering edge cases

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Fixes overtriggering edgecases for queries such as:

> Americas

> top 50

## People to notify

@bsstoner 

---

Instant Answer Page: https://duck.co/ia/view/currency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/Currency.pm [hunk 1]
```diff
@@ -175,8 +175,10 @@ handle query_lc => sub {
         return if ($amount eq '' && $to eq '' && $currencyKeyword eq '' && exists($currHash{$from}));
         
         # for edge cases that we don't want to trigger on
-        return if $req->query_lc eq 'mop tops' or $req->query_lc eq 'mop top';
-
+        return if $req->query_lc eq 'mop tops' 
+               or $req->query_lc eq 'mop top'
+               or $req->query_lc =~ m/tops?\s+?\d+/;
+
```
Comment (bsstoner) [inline lib/DDG/Spice/Currency.pm]: How does that prevent `americas` from triggering?

Comment (bsstoner) [inline lib/DDG/Spice/Currency.pm]: oh nvm, I see, you removed it below. I was so confused for a second!

**Comment by bsstoner** [comment]
👍

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**